cmd/validators/main.go

```diff
@@ -120,12 +127,159 @@ type pubKeyInfo struct {
 	Value string `json:"value"`
 }
 
+func cmdMultisigNew(out io.Writer) func() error {
+	return func() error {
+		flag.Usage = func() {
+			fmt.Fprint(flag.CommandLine.Output(), `
+Create a new validator update request with multi signature authentication.
+Created request is binary serialized and written to standard output.
+
+Returned request must be signed by other parties before it can be submitted.
+
+`)
+			flag.PrintDefaults()
```

Comment:
req: same as above to use a new `FlagSet`. Please also check the other calls.

Reply:
This is a workaround, but I like it!

I wish we could locate what is polluting our flags and fix it there.